op-e2e/actions/helpers/l2_sequencer.go

```diff
@@ -249,3 +249,11 @@ func (s *L2Sequencer) ActBuildL2ToIsthmus(t Testing) {
 		s.ActL2EmptyBlock(t)
 	}
 }
+
+func (s *L2Sequencer) ActBuildL2ToInterop(t Testing) uint64 {
```

Comment:
Why return the unsafe number here? not opposed, but from what I can tell you don't actually use it in the tests. It doesn't match the function signature of the helpers above.

Reply:
Good catch, I think this must have been an artifact of a previous approach.